packages/http-specs/specs/streaming/jsonl/main.tsp

```diff
@@ -0,0 +1,33 @@
+import "@typespec/http";
+import "@typespec/http/streams";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Http.Streams;
+using Spector;
+
+@doc("Test of jsonl streaming.")
+@scenarioService("/streaming/jsonl")
+namespace Streaming.Jsonl;
+
+@route("as-binary")
+namespace AsBinary {
```

Comment:
not sure I am a fan of `AsBinary`, its always binary. if you don't think `NoStreaming` and we can't find a better name `Basic` was ok

Reply:
just a little weird for the whole naming `Streaming.Jsonl.NoStreming`. let me switch back to `Streaming.Jsonl.Basic`.